docs/how-to/websites-on-ipfs/gateway-redirects.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Gateway redirects
```

Comment:
nit:  bit too generic, may be confusing, as we already have various  "gateway redirects" (am I redirecting from one gateway to other one? is this about redirecting from path gateway to subdomain one? etc)

My suggestion is to frame docs to be about "`_redirects` file support" or similar, this removes ambiguity, and makes it easy to find (SEO)

```suggestion
title: _redirects file support
```